tests/test_audio.py

```diff
@@ -25,25 +25,22 @@
 
 import telegram
 from tests.base import BaseTest, timeout
-from tests.bots import get_bot
 
 
 class AudioTest(BaseTest, unittest.TestCase):
     """This object represents Tests for Telegram Audio."""
 
     @classmethod
     def setUpClass(cls):
+        super().setUpClass()
```

Comment:
Oh, yeah... I did this mistake previously... Didn't learn that lesson :) Now I did ;)
Fixed.